### DISCUSSION TITLE
原画集岌岌可危

### DISCUSSION THREAD

**Discussion by czb12345** [share121/inter-knot#5459] [崩坏3]
到底谁发明出来的调休周一那天上学，等我回来原画集都没了

**Comment by Loxic123** [comment]
早十的大学生会重拳出击<img src="https://bbs-static.miyoushe.com/static/2024/09/10/d60393fe879038382e5c7cdcf7ca25fd_7267933421783671773.png" alt="飞霄 打架" width="75" >

**Comment by nostalgiatan** [comment]
我已经做好了早4早10的准备了
![img-564935084_1727438657535.jpg](https://github.com/user-attachments/assets/a164340a-e9fc-4d56-9954-6a44d5a43ea7)

**Comment by cyq20091029** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" >
呜呜呜，策划卡级了
